### PR TITLE
[wip] Initial draft, expect everything to change

### PR DESCRIPTION
**Pull request by webwurst**
/cc @puja108

### CODE REVIEW

File: docs/outlook.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+
+Not stable yet as of version 1.4:
+- [Dramatically Simplify Kubernetes Cluster Creation](https://github.com/kubernetes/features/issues/11)
```
Comment (puja108) [inline docs/outlook.md]: I would leave this out, as we kind of target K8s usage and potential customers who we don't want to create clusters themselves.

File: docs/outlook.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ScheduledJobs](https://github.com/kubernetes/features/issues/19)
+- [Configurable Dynamic Provisioning aka StorageClass](https://github.com/kubernetes/features/issues/36)
+- [Kubelet TLS Bootstrap](https://github.com/kubernetes/features/issues/43)
```
Comment (puja108) [inline docs/outlook.md]: not relevant for endusers, I think we should only include enduser relevant stuff

File: docs/outlook.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- [Federation](https://github.com/kubernetes/features/issues?q=is%3Aissue+is%3Aopen+label%3Ateam%2FSIG-Federation)
+
+Future Features:
+- [Encrypt secrets in etcd](https://github.com/kubernetes/features/issues/92)
+- [Authenticated/Authorized access to kubelet API](https://github.com/kubernetes/features/issues/89)
+- [Simplify HA Setup for Master](https://github.com/kubernetes/features/issues/48)
```
Comment (puja108) [inline docs/outlook.md]: s.o.

File: docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+## History
+
+
+
+## Community
+
+https://github.com/kubernetes/kubernetes
+
+~900 Contributors
+from Google, Red Hat, Microsoft, CoreOS, Mesosphere
```
Comment (puja108) [inline docs/overview.md]: we have a nice slide about companies at contributors and users, I'll pull that in later

**Comment by puja108** [comment]
we also want slides or something right? do we do them here or in google apps?

File: docs/outlook.md (hunk 1 above)
Comment (webwurst) [inline docs/outlook.md]: Ok, I added these to tell something more about the "ecosystem", what is Kubernetes as a project caring about and aiming for. No details, just in the outlook like as, these things might be interesting, too. So no problem if it doesn't fit or time is up or people are exhausted anyway ;)
Would like to keep it as internal note at least. Some stuff might be moved "up" to the course materiel if it fits, or stay here for the "outlook" or move out to some `knowledge-base.md`.

But you are right, some stuff here is more or less brainstorming. We could also come up with `brainstorming.md`.

**Comment by webwurst** [comment]
I would like to create slides with [`remark](https://github.com/gnab/remark). To keep it simple and also have it version controlled and reviewable. Graphics should be `svg`. Before presentation these should be uploaded to some service.

**Pull request merged**